.github/CONTRIBUTING.md

```
@@ -0,0 +1,234 @@
+# 🍸 Contributing to Cezzis Cocktails Web
+
+Thank you for your interest in contributing to the Cezzis Cocktails Web project! We welcome contributions that help improve our cocktail discovery platform.
+
+## 📋 Table of Contents
+
+- [Getting Started](#-getting-started)
+- [Development Setup](#-development-setup)
+- [Contributing Process](#-contributing-process)
+- [Code Standards](#-code-standards)
+- [Testing](#-testing)
+- [Deployment](#-deployment)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)